.github/workflows/scripts/push-scratch-images.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -e
+
+IMAGETAG="$1"
+if [ -z "$IMAGETAG" ]; then
+    echo "IMAGETAG not provided!" 1>&2
+    echo "Usage: push-images.sh IMAGETAG" 1>&2
+    exit 1
+fi
+
+# Extracting org name rather than hardcoding allows this
+# action to be portable across forks
+ORGNAME=`echo $GITHUB_REPOSITORY | tr '/' "\n" | head -1 | tr -d "\n"`
+
+echo "Pushing images tagged as $IMAGETAG..."
+
+for img in spire-server spire-agent; do
+    ghcrimg=ghcr.io/"$ORGNAME"/"$img":"${IMAGETAG}"
+    echo "Executing: docker tag $img:latest-local $ghcrimg"
+    docker tag "$img":latest-local "$ghcrimg"
```

Comment:
Shouldn't this be tagging the scratch image?
```suggestion
    docker tag "$img"-scratch:latest-local "$ghcrimg"
```

Reply:
My intention was to switch the "default" image to scratch, and that's all we ship. The move to GHCR acts as the deprecation function.

Should we continue to ship both flavors? The only argument I can see is some sort of convenience or dev image, in which case it should probably be named as such (and the non-suffixed version should still be scratch)?

Reply:
> My intention was to switch the "default" image to scratch, and that's all we ship. The move to GHCR acts as the deprecation function.

That's what I figured, but I didn't see any changes in the Makefile that changed way things are tagged and the `images.tar.gz` loaded by this step on the workflow contains the non-scratch images.

Reply:
> That's what I figured, but I didn't see any changes in the Makefile that changed way things are tagged and the images.tar.gz loaded by this step on the workflow contains the non-scratch images.

Aah yes, thanks. I will fix that.